### ISSUE TITLE
adaface loss was used to distributed training

### ISSUE THREAD

**Issue by eeric** [mk-minchul/AdaFace#13]
your adaface loss is effective, but  it was used to insightface, it occur error.
could you modified adaface loss which input was not embeddings, but logits, as like Arcface in losses.py, linked:
https://github.com/deepinsight/insightface/blob/master/recognition/arcface_torch/losses.py
Thank you first.

**Comment by mk-minchul** [comment]
Hi eeric, thanks for the suggestion. 
The overall modification is moving the FC layer out of the loss function but the function parameters also take the norm of the features as an input. 
I think I could provide the modified version in the future, but in the meanwhile I hope the users who are using the insightface code could modify the code to fit their needs :)

**Comment by eeric** [comment]
ok, I had modified afaface loss as insightface style.

**Comment by TouchSkyWf** [comment]
> ok, I had modified adaface loss as insightface style.

How did you modify the code? Can you open source the relevant code? Thanks :)
